ktor-client/ktor-client-plugins/ktor-client-platform/nonJvm/src/Platform.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.client.plugins.platform
+
+internal actual object Platform {
```

Comment:
Seems `nonJvm` source set hasn't been recognized, how can I implement this actual object for non-jvm platforms?

<img width="1197" alt="image" src="https://github.com/ktorio/ktor/assets/10363352/b5d469b2-7523-48c0-a8c7-78b13e61efd2">

Reply:
9ff34673ab4ffe5a2889fe55f1600ac93ef4129a